pkg/virt-controller/watch/workload-updater/workload-updater.go

```diff
@@ -102,8 +103,8 @@ func NewWorkloadUpdateController(
 ) (*WorkloadUpdateController, error) {
 
 	rl := workqueue.NewMaxOfRateLimiter(
-		workqueue.NewItemExponentialFailureRateLimiter(time.Duration(defaultThrottleIntervalSeconds)*time.Second, 300*time.Second),
```

Comment:
FYI, This basically means it always returned a max, of 300 Seconds.

Reply:
Right, but I'm not sure that reasoning applies to the bucket one just below, and `NewMaxOfRateLimiter()` returns the longest one.